mesh/mesh.go

```diff
@@ -3,12 +3,13 @@
 package mesh
 
 import (
-	"bytes"
 	"context"
 	"errors"
 	"fmt"
 	"sync"
 
+	"go.uber.org/atomic"
```

Comment:
Why not `sync/atomic`? It has `atomic.Value` too

Reply:
uber/atomic is just a wrapper around sync/atomic , the motivation is pretty much summarized in https://github.com/uber-go/atomic#usage